autoflip_calculators/text_detection_calculator.cc

```diff
@@ -109,21 +101,46 @@ ::mediapipe::Status TextDetectionCalculator::GetContract(
     mediapipe::CalculatorContract* cc) {
   cc->Inputs().Tag(kInputVideo).Set<ImageFrame>();
   cc->Outputs().Tag(kOutputRegion).Set<DetectionSet>();
+
   return ::mediapipe::OkStatus();
 }
 
 ::mediapipe::Status TextDetectionCalculator::Open(
     mediapipe::CalculatorContext* cc) {
-  options_ = cc->Options<TextDetectionCalculatorOptions>();
-  scorer_ = absl::make_unique<VisualScorer>(options_.scorer_options());
-  detector_ = cv::dnn::readNet(kModel_path_);
+  const auto& options = cc->Options<TextDetectionCalculatorOptions>();
```

Comment:
Optional idea: Consider making options_ itself a member variable so you don't need to copy all the settings below.